docs/releases/2.0.0-beta1.rst

```diff
@@ -46,7 +49,7 @@ Python 3 support
 - Translate Toolkit went through a massive code cleanup looking forward Python
   3 compatibility. There might be quirks that need to be fixed, so please test
   and `report any issue <https://github.com/translate/translate/issues/new>`_
-  you might find.
+  you might find. Python 3.5 is now supported.
```

Comment:
Can we specifically credit Claude for the Python 3 support within the major changes listed above.